### PR TITLE
feat(Flow): add Flow theme 

### PR DESCRIPTION
**Pull request by Ruixi-Zhang**
Flow is a new theme that removes the friend's activity and replaces it vertical playbar. It also modifies the lyrics' text size to make it more presentable. It has a gradient effect. The overall color scheme is light and pastel-like tone.
* Remove friend's activity
* Vertical Playbar
* Gradient Effect
* Change Lyrics Text Size
* Better Hovering Effect on Cards

### CODE REVIEW

File: Flow/LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2022 ossd-sp22
```
Comment (afonsojramos) [inline Flow/LICENSE:3]: Hmm, is this your name? 🤔

**Review by afonsojramos — CHANGES_REQUESTED** [review]
Looking good, I'd just request to delete the `.idea` folder if you could!

File: Flow/LICENSE (hunk 1 above)
Comment (Ruixi-Zhang) [reply Flow/LICENSE:3]: Hi, we develop this theme in a team of four. I can change the copyright name to the GitHub username of the four of us.

Comment (Ruixi-Zhang) [reply Flow/LICENSE]: Hi, I've deleted the .idea file and changed the information in the license.

**Review by afonsojramos — APPROVED** [review]

**Comment by afonsojramos** [comment]
Thanks for the new theme!!

File: Flow/README.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+### Pink
+<img src="https://raw.githubusercontent.com/ossd-sp22/spicetify-themes/newTheme/Flow/screenshots/pink.png" alt="img"> 
+
+
+### Silver
+<img src="https://github.com/ossd-sp22/spicetify-themes/blob/newTheme/Flow/screenshots/silver.png" alt="img"> 
+
+
+### Violet
+<img src="https://raw.githubusercontent.com/ossd-sp22/spicetify-themes/newTheme/Flow/screenshots/violet.png" alt="img">
```
Comment (afonsojramos) [inline Flow/README.md:18]: @Ruixi-Zhang This flew by but can you make another PR to change these to use the spicetify/spicetify-themes and the main branch as well?

**Review by afonsojramos — COMMENTED** [review]
Additionally, if you could add your theme to the manifest and to the issue assign .yml that would be awesome! Let me know if you have any trouble!

File: Flow/README.md (hunk 2 above)
Comment (Ruixi-Zhang) [reply Flow/README.md:18]: @afonsojramos The changes were updated by @ian-Liaozy
